internal/store/postgres/migrations/000005_add_permissions_to_appeal.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "appeals" DROP COLUMN IF EXISTS "permissions";
```

Comment:
If we just added a migration script, we don't need this alter table command.

Reply:
since this is for replacing auto migration, migration scripts version 1-4 are compatible sql scripts with the existing schema in the db, this version 5 is the addition to add a column

Reply:
Yes I know but Is the current migration version tracked in the DB with auto migrate?. If not we could merge the column changes into the table creation migration itself. To avoid complex migration script.

Reply:
It's actually the other way around, gorm automigrate doesn't track the schema version because it only uses the latest model and generates sql if there are any changes to the model. And with moving to go-migrate, we need to create migration scripts that match the existing schema in db first (in this case .sql version 1-4) so that when it's deployed and runs the migration script with go-migrate the first time, it won't have conflict and adds `schema_migrations` table as well.

Also, merging the column changes into the table creation won't be possible because the existing db in production already has "appeals" table, and "permissions" column needs to be added and can be done only by altering the table itself. 

The reason we're moving to go-migrate it's because we need to backfill the old data to have a proper value for the newly added "permissions" column which is not supported by gorm auto migrate

Reply:
@rahmatrhd Yes exactly that is the point. I think to make it clear (this is related with https://github.com/odpf/guardian/pull/220#discussion_r927369892).

We can have a different migration scripts PR (adding a `permission` column is not included) and merge it first. This PR could have an additional of migration script to alter table add column `permission`.